src/core/src/types/matrix.rs

```diff
@@ -91,6 +91,51 @@ impl Matrix3 {
                        [m20, m21, m22]]}
     }
 
+
+    /// Returns rotation matrix given a rotation angle and an axis.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use lumol::types::{Vector3D, Matrix3};
+    /// let e1 = Vector3D::new(1.0f64, 0.0, 0.0);
+    /// let e3 = Vector3D::new(0.0f64, 0.0, 1.0);
+    /// let angle = 90f64.to_radians();
+    ///
+    /// let rotation = Matrix3::rotation_matrix(&(e1*2.0), angle);
+    /// let mut rot_vec = rotation * e3;
+    /// for i in 0..3 {
+    ///     rot_vec[i] = (rot_vec[i] * 1.0e8).round() / 1.0e8
+    /// };
+    /// assert_eq!(rot_vec, Vector3D::new(0.0, 1.0, 0.0));
+    /// ```
+    pub fn rotation_matrix(axis: &Vector3D, angle: f64) -> Matrix3 {
```

Comment:
Should we call this function `rotation`? The matrix part is always implied now that this is a method